### ISSUE TITLE
Some strings are not i18n

### ISSUE THREAD

**Issue by 3esmit** [status-im/status-react#10842]
# Bug Report
All strings on the App should be translatable. Some strings are not translatable and hurts accessibility. 

Strings that I've found not possible to translate:

* [ ] {bio-type-label} = Fingerprint
* [ ] Notifications
* [ ] Background service notification title and text
* [ ] Sticker notification title and text
* [ ] Audio message notification title and text
* [ ] Log levels (DISABLED, ERROR, WARN, INFO, DEBUG, TRACE)
* [ ] Network information instructions: "PRESS TO REFRESH"
* [ ] Signing phrase 3 words 
* [ ] BIP39 dictionary is not using available translations
* [ ] Ethereum Network list: The list is always in English
* [ ] Users pseudo-names are always in English

# Acceptance Criteria
All items above are fixed.

#### Additional information
Build StatusIm-200616-155619-798c46-pr-universal.apk

**Comment by 3esmit** [comment]
@jinhojang6 This list is growing instead of shrinking. Now we have also the "audio messages" notification that is not translatable.

**Comment by churik** [comment]
Obsolete and will be re-considered in the redesigned app. 
Added `recheck` label for easy finding that one in case of need.
